client/app/optionsets/landuse-form.js

```diff
@@ -85,6 +85,17 @@ export const DCPDISPOSITION = {
   },
 };
 
+export const DCPLEGALINSTRUMENT = {
```

Comment:
Another YES/NO optionset from CRM 🤦‍♂️  so unfortunate
(just commenting, nothing to do here)

Reply:
ugh yep :/

Reply:
@godfreyyeung  no approval on the PR tho? :P

Reply:
Sorry!! I kept getting pulled into other stuff! 🤦‍♂️ 🙈